src/async_reader.rs

```diff
@@ -92,6 +92,11 @@ impl<B: AsyncBackend + Sync + Send> AsyncPmTilesReader<B> {
         Ok(String::from_utf8(decompressed_metadata.to_vec())?)
     }
 
+    pub async fn get_metadata_vector_layer(&self) -> Result<serde_json::Value, Error> {
```

Comment:
I'm a bit confused by the naming of this method. Could you elaborate on the name choice?

Reply:
@lseelenbinder i removed it - too many dependencies, very little value, lots of confusion. Only kept the center/bounds.